histogram/Cargo.toml

```diff
@@ -1,24 +1,16 @@
 [package]
-name = "rustcommon-histogram"
-version = "1.0.2"
-authors = ["Brian Martin <bmartin@twitter.com>"]
+name = "histogram"
+version = "0.0.0"
 edition = "2021"
-license = "Apache-2.0"
-description = "Histograms with summary metrics and precision guarantees"
-homepage = "https://github.com/twitter/rustcommon/histogram"
-repository = "https://github.com/twitter/rustcommon"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
```suggestion
```

Reply:
Fixed and added metadata like license =)